Makefile

```diff
@@ -8,7 +8,9 @@ BASE_DIR=$(CURDIR)
 ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
 # Podman DTRT by running processes unprivileged in containers,
 # but it's UID mapping is more nuanced. Only set user for vanilla docker.
-DOCKER_USER=--user "$(shell id -u)"
+DOCKER_OPTS=--user "$(shell id -u)"
+else
+DOCKER_OPTS=--security-opt label=disable
```

Comment:
I think this if-statement might be wrong..
It says when it finds the string `podman` in `docker--verison` it should add the user-id, if not disable selinux.
Locally it did not work, but when I changed the if/else statements I could build the main-image locally.

```
ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
# Disable selinux for local podman builds.
DOCKER_OPTS=--security-opt label=disable
else
# Podman DTRT by running processes unprivileged in containers,
# but it's UID mapping is more nuanced. Only set user for vanilla docker.
DOCKER_OPTS=--user "$(shell id -u)"
endif
```

Reply:
The line:

```make
ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
```

means this in english:

The output of `findstring` will be `podman` if `podman` is used, otherwise it will be empty.  If the output of `findstring` is empty, then take the first branch. Otherwise take the `else` branch.  Therefore, if podman is used, the `else` branch will be taken.

> Locally it did not work

It works for me locally.  Can you post your error?